### ISSUE TITLE
--ignore-file=x throws warnings

### ISSUE THREAD

**Issue by petdance** [petdance/ack3#69]
```
ack --ignore-file=x
Use of uninitialized value $args in split at /home/andy/bin/ack line 2406.
Unknown filter type 'x' at /home/andy/bin/ack line 2406.
```

That's in the anonymous sub for ignore-file.

If you don't pass in `--ignore-file=method:what` it should tell you that.

**Comment by petdance** [comment]
Do #70 first.

**Comment by petdance** [comment]
Fixed in 5cacd04.
